src/libraries/System.Resources.ResourceManager/tests/ResourceManagerTests.cs

```diff
@@ -82,6 +84,7 @@ public static IEnumerable<object[]> CultureResourceData()
 
         [Theory]
         [MemberData(nameof(CultureResourceData))]
+        [ActiveIssue("https://github.com/dotnet/runtimelab/issues/155", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
Which features listed in this issue is this failing on? Comment would be nice - I am not able to tell by looking at the code.